starter-code/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>REVERA Clone</title>
+  <!--  Default browser-styles resetting -->
+  <link href="thirdparty/reset.css" type="text/css" rel="stylesheet">
+  <!--  Bootstrap CSS file -->
+  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" type="text/css" rel="stylesheet">
+  <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <!--  Bootstrap JS file -->
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+  <!--  Main CSS file-->
+  <link href="main.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+  <div class="container-fluid">
+    <header>
+      <h1>REVERA</h1>
+      <ul>
+        <li>Home</li>
+        <li>About</li>
+        <li>Sample Page</li>
+        <li>Portfolio</li>
+        <li>Blog</li>
+        <li>Parent Page</li>
+        <li>Contact</li>
+      </ul>
+    </header>
+    <section class="row main-pic">
```

Comment:
This section can be done in the original style, with the revolving images, using bootstrap's carousel

https://www.w3schools.com/bootstrap/bootstrap_carousel.asp